include/swift/AST/Identifier.h

```diff
@@ -101,19 +101,22 @@ class Identifier {
   
   /// isOperator - Return true if this identifier is an operator, false if it is
   /// a normal identifier.
-  /// FIXME: We should maybe cache this.
   bool isOperator() const {
     if (empty())
       return false;
     if (isEditorPlaceholder())
       return false;
-    if ((unsigned char)Pointer[0] < 0x80)
-      return isOperatorStartCodePoint((unsigned char)Pointer[0]);
 
     // Handle the high unicode case out of line.
     return isOperatorSlow();
   }
 
+  /// Returns true if this identifier contains non-identifier characters and
+  /// must always be escaped with backticks, even in contexts were other
+  /// escaped identifiers could omit backticks (like keywords as argument
+  /// labels).
+  bool mustAlwaysBeEscaped() const;
```

Comment:
Could you add some test cases for code completion checking  `mustAlwaysBeEscaped()` items are actually escaped in code completion suggestion?

Reply:
Thanks! Those did need to be fixed.